src/main/kotlin/com/atlassian/performance/tools/virtualusers/api/users/UserGenerator.kt

```diff
@@ -4,6 +4,11 @@ import com.atlassian.performance.tools.jiraactions.api.memories.User
 import com.atlassian.performance.tools.virtualusers.api.VirtualUserOptions
 import net.jcip.annotations.ThreadSafe
 
+/**
+ * Supplies a user. The user can be created dynamically or it can find an existing user.
```

Comment:
So `UserGenerator#generateUser` name is a little bit misleading. At least docs explain it now. 👍

Reply:
True that. It should be called `UsersSource`, `UserSupplier` or something like that.